code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -226,6 +226,11 @@ datum/reagent/medicine/adminordrazine/on_mob_life(var/mob/living/carbon/M as mob
 	..()
 	return
 
+datum/reagent/medicine/adminordrazine/nanites
```

Comment:
It still technically isnt nerfed from the level of stuff that the actual admin chem does.
https://github.com/tgstation/-tg-station/blob/c49c853bf82f46470940fa9cd7b5dff270455297/code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm#L196
The amount of healing this will do pretty much anyone will be instantly healed from ANY issue.
Please please please just make another medicine outside of this damn admin one that heals most things but it really should not heal dead people.